.github/workflows/topos:integration-tests.yml

```diff
@@ -165,6 +165,20 @@ jobs:
           ERC20_MESSAGING_SALT: ${{ secrets.ERC20_MESSAGING_SALT }}
           SUBNET_REGISTRATOR_SALT: ${{ secrets.SUBNET_REGISTRATOR_SALT }}
 
+      - name: Run target sequencer cert delivery test
```

Comment:
I would change the order of this test to before `test_cert_inclusion.sh` as we want the cert to be seen by the sequencer first and then the subnet related to that sequencer.